ci/unit_tests/workspace_addjson/main_test.py

```diff
@@ -0,0 +1,32 @@
+
+import os, unittest, sys, codecs, json
+
+import workspace_addjson
+
+class TestMain(unittest.TestCase):
+
+    ''' Tests if particular file is found by getFilesAtPath function '''
+    def test_main(self):
+
+        cmd = 'cp ./tests/test_data/workspace.json ./tests/test_data/workspace_forAddJsonTest.json'
+        os.system(cmd)
+
+        workspaceFile = 'workspace_forAddJsonTest.json'
+        directory = './tests/test_data/'
+        jsonAddPath = './tests/test_data/jsonToAdd.json'
+        targetNode = 'parent'
+        referenceResult = './tests/test_data/workspace_result.json'
```

Comment:
Data for this test should be placed in its folder (i.e. `ci/unit_tests/workspace_addjson`).

Reply:
done